src/styles/buttons.ts

```diff
@@ -121,8 +122,22 @@ export const card: ViewStyle = {
 
 export const fixedBottom: ViewStyle = {
   ...base,
-  paddingTop: Spacing.medium,
-  paddingBottom: Spacing.medium,
+  paddingVertical: Spacing.medium,
   backgroundColor: Colors.primary.shade100,
   width: "100%",
 }
+
+export const fixedBottomDisabled: ViewStyle = {
```

Comment:
:100:
i wonder if you could do the nested structure here? something like:
```typescript
Buttons.fixedBottom.disabled
Buttons.fixedBottom.thin
etc...
```

Reply:
That's the plan! 😄